Advice on hnix dependency

I'm trying to use hocker via Nix's `fetchFromGitHub` and `cabal2Nix` but seeing `error: Package 'hnix-0.5.2' in /nix/store/<hash>-source/pkgs/development/haskell-modules/hackage-packages.nix:<line> is marked as broken, refusing to evaluate.`

If I mark it as unbroken, I run into:

```
[12 of 35] Compiling Nix.Expr.Types.Annotated ( src/Nix/Expr/Types/Annotated.hs, dist/build/Nix/Expr/Types/Annotated.o )
```

```
src/Nix/Render.hs:26:17: error:
    • Couldn't match expected type ‘Int’
                  with actual type ‘NonEmpty SourcePos’
    • In the first argument of ‘FancyError’, namely ‘(beg :| [])’
      In the expression:
        FancyError
          (beg :| [])
          (Set.fromList [ErrorFail (show msg) :: ErrorFancy Void])
      In an equation for ‘posAndMsg’:
          posAndMsg beg msg
            = FancyError
                (beg :| [])
                (Set.fromList [ErrorFail (show msg) :: ErrorFancy Void])
   |
26 |     FancyError (beg :| [])
   |                 ^^^^^^^^^

src/Nix/Render.hs:35:28: error:
    • Variable not in scope:
        parseErrorPretty' :: ByteString -> ParseError t0 Void -> [Char]
    • Perhaps you meant ‘parseErrorPretty’ (imported from Text.Megaparsec.Error)
   |
35 |     return $ text $ init $ parseErrorPretty' contents (posAndMsg beg msg)
   |                            ^^^^^^^^^^^^^^^^^
builder for '/nix/store/4si6chlwafprv3fb70pb6sz9f8rk70m3-hnix-0.5.2.drv' failed with exit code 1
cannot build derivation '/nix/store/xvk74c2xb7xrbq8mckcyjsv3qc5j7nvc-hocker-1.0.5.drv': 1 dependencies couldn't be built
error: build of '/nix/store/xvk74c2xb7xrbq8mckcyjsv3qc5j7nvc-hocker-1.0.5.drv' failed
```

Any suggestions?

--- a/hocker.cabal
+++ b/hocker.cabal
@@ -79,7 +79,7 @@ library
                 exceptions           >= 0.8,
                 filepath             >= 1.4,
                 foldl                >= 1.0,
-                hnix                 >= 0.5.2,
+                hnix                 >= 0.6.0,
                 http-client          >= 0.4,
                 http-types           >= 0.9.1,
                 lens                 >= 4.0,

